.github/workflows/analyse-pr.yml

```diff
@@ -0,0 +1,47 @@
+name: SonarQube analysis
+
+on:
+  push:
+    branches:
+    - master
+    - '2.3[1-9]'
```

Comment:
Is this a regexp? Could we use e.g. `\d\.\d+` instead? Otherwise we have to remember to change this in the future. It is probable that we forget to do so.

Reply:
I didn't want to include all the unsupported branches we have. But could make it last longer than 5 future versions, yeah :D

Reply:
Ah, I see. :)